README.md

```diff
@@ -46,18 +46,18 @@ In this dataset, the data is available from 1999 for the oldest, to 2019/12/31 f
     - `forecast_time`: from 2000/01/02 to 2019/12/31, weekly every 7 days (every Thurday).
     - `lead_time`: 0 to 46 days
     - `valid_time` (`forecast_time` + `lead_time`): from 2000/01/02 to 2020/02/13
-    - availables parameters : `2t/ci/gh/lsm/msl/q/rsn/sm100/sm20/sp/sst/st100/st20/t/tcc/tcw/tp/ttr/u/v`
+    - availables parameters : `t2m/ci/gh/lsm/msl/q/rsn/sm100/sm20/sp/sst/st100/st20/t/tcc/tcw/tp/ttr/u/v`
```

Comment:
```suggestion
    - availables parameters : `t2m(2t)/ci(siconc)/gh/lsm/msl/q/rsn/sm100/sm20/sp/sst/st100/st20/t/tcc/tcw/tp/ttr/u/v` (differing name in [MARS database](https://confluence.ecmwf.int/display/S2S/Parameters))
```

Reply:
does this look plausible to you @floriankrb ?

Reply:
yes, this looks good to me, thanks !

Reply:
so either ci(MARS) or t2m(CF), currently in cloud, needs to be renamed whatever we do. I prefer to use the CF name because I do not want to rename t2m on website, in all examples etc.

Reply:
ok. should I wait with merging this until the code is ready for that? @floriankrb